gitlab/v4/objects/job_token_scope.py

```diff
@@ -0,0 +1,23 @@
+from typing import Any, cast
+
+from gitlab.base import RESTManager, RESTObject
+from gitlab.mixins import GetWithoutIdMixin, RefreshMixin, SaveMixin, UpdateMixin
+
+__all__ = [
+    "ProjectJobTokenScope",
+    "ProjectJobTokenScopeManager",
+]
+
+
+class ProjectJobTokenScope(RefreshMixin, SaveMixin, RESTObject):
+    _id_attr = None
+
+
+class ProjectJobTokenScopeManager(GetWithoutIdMixin, UpdateMixin, RESTManager):
+    _path = "/projects/{project_id}/job_token_scope"
+    _obj_cls = ProjectJobTokenScope
+    _from_parent_attrs = {"project_id": "id"}
+    _update_uses_patch = True
```

Comment:
I think this needs to be updated.

Reply:
Sorry that was a lazy rebase :grin: fixed.